onadata/libs/utils/chart_tools.py

```diff
@@ -208,6 +208,10 @@ def build_chart_data_for_field(xform, field, language_index=0, choices=None,
         field_label = 'Submission Time'
         field_xpath = '_submission_time'
         field_type = 'datetime'
+    elif isinstance(field, basestring) and field == common_tags.SUBMITTED_BY:
```

Comment:
I think this is a pattern already laid out before, see most of the if statements already in the file, also a field tends to be an object most of the time.